src/server/httpcommon.cpp

```diff
@@ -20,17 +20,17 @@
 #include <boost/asio/ssl/context_base.hpp>
 #include <curl/curl.h>
 
-#include "config.h"
 #include "crypto.h"
-#include "file_handler.h"
 #include "httpcommon.h"
```

Comment:
Do these imports work if you do `src/server/httpcommon.h` for example? It might be better to be more explicit?